internal/cltest/cltest.go

```diff
@@ -613,16 +613,25 @@ func CreateJobRunViaWeb(t *testing.T, app *TestApplication, j models.JobSpec, bo
 
 // CreateHelloWorldJobViaWeb creates a HelloWorld JobSpec with the given MockServer Url
 func CreateHelloWorldJobViaWeb(t *testing.T, app *TestApplication, url string) models.JobSpec {
-	j := FixtureCreateJobViaWeb(t, app, "../internal/fixtures/web/hello_world_job.json")
-	j.Tasks[0].Params = JSONFromString(fmt.Sprintf(`{"url":"%v"}`, url))
-	return CreateJobSpecViaWeb(t, app, j)
+	buffer, err := ioutil.ReadFile("../internal/fixtures/web/hello_world_job.json")
+	if err != nil {
+		assert.FailNowf(t, "Unable to read fixture", err.Error())
+	}
+
+	var job models.JobSpec
+	err = json.Unmarshal(buffer, &job)
+	require.NoError(t, err)
+
+	job.ID = utils.NewBytes32ID()
```

Comment:
I think we can ✂️ cut this line. The `ID` gets populated server side when the spec is created.

Reply:
It does not work without this, digging into why...

Reply:
Fixed.